frontend/src/components/organisms/EventCard.tsx

```diff
@@ -13,11 +13,11 @@ import { Action } from "@/utils/types";
 import { formatDateTimeRange } from "@/utils/helpers";
 
 interface EventCardProps {
-  eventid: string;
+  eventid: string | undefined;
```

Comment:
Replaced with `eventid?; string;` because it's functionally equivalent and cleaner.